app/assets/javascripts/discourse/app/instance-initializers/category-badge-css-generator.js

```diff
@@ -17,9 +17,9 @@ export default {
       let parentCategory = Category.findById(
         get(category, "parent_category_id")
       );
-      let badgeCss = `.badge-category[data-category-id="${category.id}"] { --category-badge-color: var(--category-${category.id}-color); --category-badge-text-color: #${category.text_color}; }`;
```

Comment:
I am wondering if this change is required as part of the fix. If not, I think we should split it out as a UX change in another PR>

Reply:
Hmm, I think I can move some of these to a different PR.